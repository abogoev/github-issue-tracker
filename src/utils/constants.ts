export const ACTIVE_OPACITY = 0.8;

export const ISSUE_SEARCH_PARAMS = "issueSearchParams";

export const REPO = "repo";
export const OWNER = "owner";
export const STATE = "state";
