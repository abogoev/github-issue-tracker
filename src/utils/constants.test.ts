import * as constants from "./constants";

describe("constants", () => {
  it("should match snapshot", () => {
    expect(constants).toMatchSnapshot();
  });
});
