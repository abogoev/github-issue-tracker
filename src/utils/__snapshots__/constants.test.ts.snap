// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should match snapshot 1`] = `
Object {
  "ACTIVE_OPACITY": 0.8,
  "ALL": "all",
  "BOOKMARKS_SCREEN": "Bookmarks",
  "CLOSED": "closed",
  "DETAILS_SCREEN": "Details",
  "HOME_SCREEN": "Home",
  "ISSUE_SEARCH_PARAMS": "issueSearchParams",
  "OPEN": "open",
  "OWNER": "owner",
  "REPO": "repo",
  "STATE": "state",
}
`;
